src/lang/nodejs/package.json

```diff
@@ -18,17 +19,23 @@
   "homepage": "https://github.com/arsalanc-v2/blotjson#readme",
   "dependencies": {
     "open": "^7.0.4",
-    "websocket": "^1.0.31",
-    "ws": "^7.3.0"
+    "websocket": "^1.0.31"
   },
   "devDependencies": {
+    "ws": "^7.3.0",
     "eslint": "^7.1.0",
     "eslint-config-standard": "^14.1.1",
     "eslint-plugin-import": "^2.20.2",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-standard": "^4.0.1",
+    "husky": "^4.2.5",
     "jest": "^26.0.1",
     "prettier": "2.0.5"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npx prettier --check src/*.js && npx eslint *src/index.js"
```

Comment:
I think using the `--write` option for prettier instead would be better so the code can be formatted directly.

Reply:
just a tiny issue with this is that the changed files will have to be commited again since after they were changed, the latest files were no longer in the staging area

Reply:
Alright we can leave it as is then.